chickadee/processes/wps_QDM.py

```diff
@@ -1,21 +1,19 @@
 from pywps import Process
 from pywps.app.Common import Metadata
-from wps_tools.utils import log_handler
+from wps_tools.utils import log_handler, collect_args, common_status_percentages
 from wps_tools.io import log_level, nc_output
 from chickadee.utils import (
     logger,
     get_package,
-    collect_args,
     set_end_date,
-    common_status_percentage,
 )
 from chickadee.io import gcm_file, obs_file, varname, out_file, num_cores, end_date
 
 
 class QDM(Process):
     def __init__(self):
         self.status_percentage_steps = dict(
-            common_status_percentage,
+            common_status_percentages,
             **{"get_ClimDown": 5, "parallelization": 10, "set_end_date": 15},
```

Comment:
QDM doesn't actaully use the `end_date` option. You can see in the [config file](https://github.com/pacificclimate/ClimDown/blob/master/R/config.R#L5) for ClimDown which options it uses. I wasn't sure until I started working on the options branch and looked at that file.